public/less/take-a-stand.less

```diff
@@ -99,66 +99,135 @@
 
 // new TAS using these selectors
 
-div.standings {
-  margin: auto;
+.banner {
+  background-color: #F6B316;
+  text-align: center;
 
-  .meter {
-    margin: auto;
+  div.standings{
     width: 50%;
-    height: 1em;  /* Can be anything */
-    position: relative;
-    background: #555;
-    -moz-border-radius: 25px;
-    -webkit-border-radius: 25px;
-    border-radius: 25px;
-    padding: 10px;
-    box-shadow: inset 0 -1px 1px rgba(255,255,255,0.3);
-
-    span {
-      display: block;
-      height: 100%;
-      border-top-right-radius: 8px;
-      border-bottom-right-radius: 8px;
-      border-top-left-radius: 20px;
-      border-bottom-left-radius: 20px;
-      background-color: #F6B316;
-      background-image: linear-gradient(
-        center bottom,
-        rgb(43,194,83) 37%,
-        rgb(84,240,84) 69%
-      );
-      box-shadow:
-        inset 0 2px 9px  rgba(255,255,255,0.3),
-        inset 0 -2px 6px rgba(0,0,0,0.4);
+    margin: auto;
+    text-align: center;
+
+    div span {
```

Comment:
It's better to avoid tag selectors in css whenever possible. It has to check this selector for every single span on the page. If we can give it a class in the html it will run much faster.